libraries/usersearch/api/src/main/kotlin/io/element/android/libraries/usersearch/api/UserSearchResult.kt

```diff
@@ -22,3 +22,8 @@ data class UserSearchResult(
     val matrixUser: MatrixUser,
     val isUnresolved: Boolean = false,
 )
+
+data class UserSearchResults(
```

Comment:
Maybe rename it to `UserSearchResultState`? It's quite easy to mistake `UserSearchResult` with `UserSearchResults`.

Reply:
It was what I had previously, but it's not really a `State` neither...